samples/file-encryptor/enclave/encryptor.cpp

```diff
@@ -64,7 +47,8 @@ int ecall_dispatcher::initialize(
         goto exit;
     }
     // init iv
-    memcpy(m_operating_iv, m_original_iv, IV_SIZE);
+    memcpy(m_original_iv, header->salt, SALT_SIZE_IN_BYTES);
```

Comment:
Is it necessary to keep the extra `m_original_iv` buffer anymore? It looks like the encryptor is always re-initialized given the header anyway, the `m_original_iv` really isn't used anywhere.

Reply:
Removed m_original_iv